backend/server.js

```diff
@@ -30,9 +30,15 @@ app.get("/working-hours", (req, res) => {
   );
 });
 
-app.get("/monthly-working-hours", (req, res) => {
+app.get("/monthly-working-hours/:year/:amount", (req, res) => {
+  let year = req.params.year;
+  const amount = req.params.amount;
```

Comment:
Something like `yearsBack` or `previousYears` would be a more descriptive variable name. Amount doesn't tell what it counts. Bananas? Years?